### ISSUE TITLE
Midterm Q1(d)

### ISSUE THREAD

**Issue by BobShoaun** [dilevin/computer-graphics-csc317#34]
Do we need to provide the name of the porter-duff operator and blend mode?

Does "label all sections of compositing table with the image source of their colours" mean if the color is red, write "Source", and if blue write "Destination" etc?

Do we also have to provide mathematical formula of the composition?

**Comment by dilevin** [comment]
No name required 

yes label table with appropriate inputs

**Comment by BobShoaun** [comment]
what about transparent? there is no image source that is fully transparent

**Comment by dilevin** [comment]
Checkerboard pattern on images indicates transparency 

You have enough information in the inputs to answer the question provided you recall how porter duff compositing works from class.

**Comment by Caspian-9** [comment]
Do we have to write the mathematical formulas for the composition?

**Comment by dilevin** [comment]
No. The question clearly specifies you only need to fill in the table
